scripts/pin-ginkgo-version.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# fail if some commands fails
+set -e
+# show commands
+set -x
+
+CHECKOUT_GINKGO_VERSION="v1.14.0"
+
+# Delete existing ginkgo repo
+if [ -d "$GOPATH/src/github.com/onsi/ginkgo" ]; then
+    rm -rf $GOPATH/src/github.com/onsi/ginkgo
+fi
+
+git clone https://github.com/onsi/ginkgo $GOPATH/src/github.com/onsi/ginkgo
+pushd $GOPATH/src/github.com/onsi/ginkgo
+git checkout $CHECKOUT_GINKGO_VERSION
+go install -mod=“” github.com/onsi/ginkgo/ginkgo
```

Comment:
use normal double quotes and it should work 

```suggestion
go install -mod="" github.com/onsi/ginkgo/ginkgo
```